.github/workflows/ci.yml

```diff
@@ -50,4 +50,40 @@ jobs:
       - uses: julia-actions/julia-processcoverage@v1
       - uses: codecov/codecov-action@v1
         with:
-          file: lcov.info
\ No newline at end of file
+          file: lcov.info
+  
+  docs:
+    name: Documentation
+    runs-on: ubuntu-latest
+    steps:
+      # Cancel ongoing documentation build if pushing to branch again before the previous 
+      # build is finished.
+      - name: Cancel ongoing documentation builds for previous commits
+        uses: styfle/cancel-workflow-action@0.6.0
+        with:
+          access_token: ${{ github.token }}
+      
+      # Build docs
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@v1
+        with:
+          version: '1'
+      - name: Install dependencies
+        run: |
+          julia --project=docs -e '
+            using Pkg
+            Pkg.develop(PackageSpec(path=pwd()))
+            Pkg.instantiate()
+            '
+      - name: Generate documentation and deploy
+        env: # needed for pushing to gh-pages branch
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }}
+        run:
+          julia --project=docs docs/make.jl
+      - name: Doctests
+        run: |
+          julia --project=docs -e '
+            using Documenter: doctest
+            using TimeseriesSurrogates
+            doctest(TimeseriesSurrogates)'
```

Comment:
Kinda useless step, we don't have any doctests in the documentation strings to be performed. And I certainly don't see the usage of them, as we have proper documentation hosted anyways.